src/bin/slackApi.ts

```diff
@@ -71,26 +71,39 @@ async function getTextInternal(aiClient: AIClient, channel, input, reminders) {
 
 const SUPER_CLUSTER_USER_STRING = 'UVBBEEC4A';
 
-async function sendOotpChat(messages) {
-  axios
-    .post(
-      'https://ootp.bedaire.com/chat',
-      {
-        context: {
-          bot: SUPER_CLUSTER_USER_STRING,
-        },
-        messages,
+async function sendOotpChat(messages, channel, say) {
+  const response = await axios.post(
+    'https://ootp.bedaire.com/chat',
+    {
+      context: {
+        bot: SUPER_CLUSTER_USER_STRING,
       },
-      {
-        headers: { 'Content-Type': 'application/json' },
-      }
-    )
-    .then((response) => {
-      console.log('chat response', response.data);
-    })
-    .catch((error) => {
-      console.log('chat error', error);
-    });
+      messages,
+    },
+    {
+      headers: { 'Content-Type': 'application/json' },
```

Comment:
Might be able to drop this?

Reply:
I think you're right, per https://axios-http.com/docs/post_example.